scripts/config-overrides/custom_start.js

```diff
@@ -4,16 +4,69 @@ const { spawn } = require('child_process');
 const port = process.env.PORT || 4321;
 process.env.PORT = port;
 
-const react_script_start = 'npx react-scripts start';
-const react_app_rewired_start = 'npx react-app-rewired start --config-overrides=scripts/config-overrides';
+const reactAppRewiredStart =
+  'npx react-app-rewired start --config-overrides=scripts/config-overrides/';
 
-if (process.env.ALLOW_LOGS === "YES") {
-  // Execute the npm command
```

Comment:
Why was this logic removed?

Reply:
Hey @palisadoes I think it was removed by mistake, it should be present there.

Reply:
The logic is still removed. Why?